XamarinCommunityToolkit/Extensions/TranslateExtension.shared.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using Xamarin.CommunityToolkit.Helpers;
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+namespace Xamarin.CommunityToolkit.Extensions
+{
+    [ContentProperty(nameof(Text))]
+    public class TranslateExtension : IMarkupExtension<BindingBase>
+    {
+        public string Text { get; set; }
+        public string StringFormat { get; set; }
+        object IMarkupExtension.ProvideValue(IServiceProvider serviceProvider) => ProvideValue(serviceProvider);
+
+        public BindingBase ProvideValue(IServiceProvider serviceProvider)
+        {
+        #if !NETSTANDARD1_0
+            var binding = new Binding
+            {
+                Mode = BindingMode.OneWay,
+                Path = $"[{Text}]",
+                Source = LocalizationResourceManager.Current,
+                StringFormat = StringFormat
+            };
+            return binding;
+        #else
+           throw new NotSupportedException(); 
```

Comment:
So we're not supporting netstandard1 at all with this feature?

Reply:
Yes! This feature will be only available for Netstandard >= 2.

Reply:
Is it simply not possible at all? Should we investigate? Although I don't think it will be a big problem

Reply:
The problem is that **System.Threading** is not available in Netstandard < 2. https://github.com/dotnet/standard/issues/34  
This why I created this plugin I time ago https://github.com/CrossGeeks/MultilingualPlugin :P